mmcv/runner/iter_based_runner.py

```diff
@@ -206,7 +166,7 @@ def save_checkpoint(self,
             # meta.update(epoch=self.epoch + 1, iter=self.iter) otherwise
             # there will be problems with resumed checkpoints.
             # More details in https://github.com/open-mmlab/mmcv/pull/1108
-        meta.update(epoch=self.epoch + 1, iter=self.iter)
+        meta.update(epoch=self.epoch + 1, iter=self.iter + 1)
```

Comment:
In most cases, the epoch should be `self.epoch` instead of `self.epoch + 1`?

Reply:
No, when iterating the i-th epoch, the `self.epoch` is `i-1`.

Reply:
In the resume code, the `self.epoch` is resumed directly from the checkpoint. If the iterbased runner resumed multiple times in the middle of one epoch, the epoch number will be wrong.

Reply:
certainly